### PR TITLE
'Immediately executed' -> 'Immediately invoked'

### PR DESCRIPTION
**Pull request by davidmaxwaterman**
IIFE is the usual term for these things (as mentioned above the edit). Using another term might lead the reader to think they are different things.

> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

> Anything else that could help us review it

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
I agree!

File: files/en-us/web/javascript/reference/operators/function/index.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ button.addEventListener('click', function(event) {
 })
 ```
 
-### Using an immediately executed function expression
+### Using an Immediately Invoked Function Expression ([IIFE (Immediately Invoked Function Expression)](/en-US/docs/Glossary/IIFE))
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/operators/function/index.md:139]: Sorry I mis-looked at it.

We cannot add a link in the headings.

Can you add a sentence right after it, that links to the IIFE glossary entry?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
We need not to put a link in the heading.

File: files/en-us/web/javascript/reference/operators/function/index.md (hunk 1 above)
Comment (davidmaxwaterman) [reply files/en-us/web/javascript/reference/operators/function/index.md:139]: No real need. Yes, glossary - it's the same link as in the IIFE reference higher up the page. I'll just make it plain text, if I can figure out how to do that.

**Review by teoli2003 — APPROVED** [review]
Yes, it did! Thanks.

**Pull request merged**